clang/test/SemaSYCL/sycl-dllimport-dllexport.cpp

```diff
@@ -0,0 +1,50 @@
+// RUN: %clang_cc1 -triple x86_64-pc-linux-gnu -fsycl-is-device -fms-extensions -DTEST %s -o /dev/null 2>&1 | FileCheck %s --check-prefixes CHECK
```

Comment:
Is this check needed at all? '-triple x86_64-pc-linux-gnu -fsycl-is-device' seems odd to me. (I may be wrong though).

Reply:
I intent to test this.  For non-window OS, warning should be emit.

Reply:
Ok, did you use x86_64 instead of spir64 intentionally?
I mean, for the same purpose you could use spir64-unknown-linux-sycldevice, which has more sense when used with "-fsycl".